src/localstorage.js

```diff
@@ -0,0 +1,38 @@
+// localstorage.js
+export default class ToDo {
+  constructor() {
+    const storedData = JSON.parse(localStorage.getItem('ToDo-storage'));
+    this.taskList = Array.isArray(storedData) ? storedData : [];
+  }
+
+  // add function
+  addToDo(description) {
+    const completed = false;
+    const index = this.taskList.length;
+    const updatedToDo = [
+      ...this.taskList,
+      { completed, description, index },
+    ];
+    this.updateStorage(updatedToDo);
+  }
```

Comment:
- [ ] Kindly note that `index` values of the tasks should correspond with their locations in the array starting from 1. So the first tasks should have an index of 1, and the second one should be 2 and so on. But currently your `index` start at 0. Kindly fix that.

Reply:
@TedLivist Thank you so much for pointing this out.